flink-table-store-format/src/main/java/org/apache/flink/table/store/format/parquet/ParquetFileFormatFactory.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.table.store.format.parquet;
+
+import org.apache.flink.configuration.Configuration;
+import org.apache.flink.table.store.format.FileFormatFactory;
+
+import java.util.Properties;
+
+/** Factory to create {@link ParquetFileFormat}. */
+public class ParquetFileFormatFactory implements FileFormatFactory {
+    public static final String IDENTIFIER = "parquet";
+
+    @Override
+    public String identifier() {
+        return IDENTIFIER;
+    }
+
+    @Override
+    public ParquetFileFormat create(Configuration formatOptions) {
+        return new ParquetFileFormat(supplyDefaultOptions(formatOptions));
+    }
+
+    private Configuration supplyDefaultOptions(Configuration options) {
+        if (!options.containsKey("compress")) {
```

Comment:
It is `compression` for parquet

Reply:
You can `ParquetOutputFormat.COMPRESSION`, and remove `parquet.` prefix.

Reply:
Thanks for your reminds, I have fixed it.

Reply:
CI failed caused by "native lz4 library not available", i'm inspecting the problem :)

Reply:
I found that the codes in flink-parquet module has showed that default compression is snappy, given the consistency experience of users, maybe we could make snappy default compression in parquet formats. 
```
FlinkParquetBuilder#createWriter()
return new ParquetRowDataBuilder(out, rowType, utcTimestamp)
                    .withCompressionCodec(
                            CompressionCodecName.fromConf(
                                    conf.get(
                                            ParquetOutputFormat.COMPRESSION,
                                            CompressionCodecName.SNAPPY.name())))
```